core/src/main/java/com/squarespace/template/plugins/platform/CommerceFormatters.java

```diff
@@ -493,14 +493,14 @@ public void apply(Context ctx, Arguments args, Variables variables) throws CodeE
       JsonNode pricingOptions = CommerceUtils.getPricingOptionsAmongLowestVariant(node);
 
       if (pricingOptions != null && pricingOptions.size() > 0) {
-        if (CommerceUtils.hasVariants(node)) {
+        if (CommerceUtils.hasVariedPrices(node)) {
```

Comment:
Instead of just checking if there's variants, we need to also check if there's variants and the prices are different among them.